ironfish/src/telemetry/telemetry.ts

```diff
@@ -281,9 +281,18 @@ export class Telemetry {
   }
 
   submitNodeStarted(): void {
+    let fields: Field[] = [{ name: 'online', type: 'boolean', value: true }]
+
+    if (this.metrics) {
+      fields = fields.concat([
+        { name: 'cpu_cores', type: 'integer', value: this.metrics.cpuCores },
+        { name: 'total_mem', type: 'integer', value: this.metrics.memTotal },
```

Comment:
nit: Kind of weird it's `total_mem` and the metrics field is `memTotal`. What do you think about `memory_total` to be explicit / consistent?

Reply:
Yes please, I agree with Rohan.